README.md

```diff
@@ -1 +1,179 @@
-# Budget-app-
\ No newline at end of file
+<h1>My_Wallett</h1>
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Video-Demo](Video-Demo)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [My_Wallett] <a name="about-project"></a>
+
+**[My_Wallett]**
+ this is mobile web application designed for budget management. The app enables users to handle their finances efficiently by organizing transactions linked to specific categories. This functionality offers users visibility into their spending patterns, allowing them to track expenses and understand where their money is allocated.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+     <li><a href="https://developer.mozilla.org/en-US/docs/Web/css">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Models]**
+- **[Controllers]**
+- **[Views]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git git@github.com:Nessrine88/My_Wallet.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec/models/model_name_spec.rb
+```
+
+### Deployment
+
+You can deploy this project using:
+Coming Soon
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+### Video-Demo
+ [Video-demo](https://www.loom.com/share/3b21c503d46b4857a006111257ab3f65?sid=67dd9a81-3057-4efa-b361-8810f8a4089d)
```

Comment:
- [ ] It will be nice to also capture which aspect of the code you find interesting in your video presentation.